setup.py

```diff
@@ -13,6 +13,7 @@
             'kibicara_mda=kibicara.platforms.email.mda:Main',
         ]
     },
+    python_requires='>=3.8',
```

Comment:
see https://packaging.python.org/guides/dropping-older-python-versions/#specify-the-version-ranges-for-supported-python-distributions for documentation and https://www.python.org/dev/peps/pep-0440/#version-specifiers for the syntax